app/packages/core/src/components/ColorModal/colorPalette/Colorscale.tsx

```diff
@@ -284,5 +293,5 @@ export default Colorscale;
 
 const validateFloat = (n: number) => {
```

Comment:
nit: this is more specific than float validation, I believe `validateUnitInterval` may be an apt name